docs/mobile/events.md

```diff
@@ -21,46 +20,47 @@ application lifecycle. This event is meant to be used in conjunction with
 `os.name` [resource semantic convention](/docs/resource/os.md) to identify the
 mobile operating system (e.g. Android, iOS).
 
-### iOS
+The following table describes the payload fields that MUST
+be used to describe the state of the application at the time of the event.
 
-<!-- semconv ios.lifecycle.events -->
-The event name MUST be `device.app.lifecycle`.
-
-| Attribute  | Type | Description  | Examples  | Requirement Level |
-|---|---|---|---|---|
-| `ios.state` | string | This attribute represents the state the application has transitioned into at the occurrence of the event. [1] | `active` | Required |
+The `android.state` and `ios.state` fields are mutually exclusive and MUST
+NOT be used together, each field MUST be used with its corresponding
+`os.name` [resource semantic convention](/docs/resource/os.md) value.
 
-**[1]:** The iOS lifecycle states are defined in the [UIApplicationDelegate documentation](https://developer.apple.com/documentation/uikit/uiapplicationdelegate#1656902), and from which the `OS terminology` column values are derived.
+### Event details
 
-`ios.state` MUST be one of the following:
+<!-- semconv device.app.lifecycle(full) -->
+The event name MUST be `device.app.lifecycle`.
 
```

Comment:
Can we document the severity level for this event since it's part of Event API and important for filtering?

Reply:
Severity is not in the payload though, and instrumentations can log the event at any severity level.

Reply:
There is no severity associated with the event, and severity is an Optional field and is not required for this event.